code/_onclick/observer.dm

```diff
@@ -14,6 +14,10 @@
 	if(..())
 		return TRUE
 
+	// If the player is auto-observing someone, and they clicked on part of the target's UI.
+	if((observe_target_client || observe_target_mob) && istype(target, /atom/movable/screen))
```

Comment:
Is this necccessary? Won't it prevent using observer jump alerts while following someone?